ODLA/platforms/vodla/CMakeLists.txt

```diff
@@ -1,3 +1,7 @@
+add_definitions(-DTIMING)
+add_definitions(-DLOOP_CNT=1)
+set(CMAKE_CXX_FLAGS_DEBUG "${CMAKE_CXX_FLAGS_DEBUG} -DDEBUG -g3")
```

Comment:
no need for this. 
you can do "cmake -DCMAKE_BUILD_TYPE=DEBUG"

Reply:
Okay, deleted now.